### DISCUSSION TITLE
Drucker Prager model

### DISCUSSION THREAD

**Discussion by pipiqiu311** [DualSPHysics/DualSPHysics#203] [DUALSPHYSICS_V5.2]
Hello, we want to investigate soil in DualSPHysics with the Drucker Prager model. Could you please provide some suggestions where to start?

**Comment by alexbexe** [comment]
https://github.com/DualSPHysics/DualSPHysics/wiki/3.-SPH-formulation#317-rheology-models-and-non-newtonian-formulations

**Comment by lijunhao0218** [comment]
Hello, I am also in the area of geotechnical simulation and your question is also my concern, I wonder if you have solved the above problem, if you can, I would very much appreciate if you could give me some tips and answers, thanks!

**Comment by yzhao469** [comment]
Hi guys, I know two DualSPHysics based soil model options:
1. GeoDualSPHysics (https://www.sciencedirect.com/science/article/pii/S0010465525004667)
2. DualSPhysics-INL (https://github.com/IdahoLabResearch/DualSPHysics-INL) This is what I developed
